example/phantom-example/index.js

```diff
@@ -42,7 +40,7 @@ crawler.emit = function(name, queueItem) {
         return string;
     }
 
-    if (!~boringEvents.indexOf(name)) {
+    if (boringEvents.indexOf(name) === -1) {
```

Comment:
Thanks for fixing this! :)

Next time I'll try to do it myself so I don't have to bother you, haha.